localsettings.example.py

```diff
@@ -12,6 +12,11 @@
     }
 }
 
+# Custom databases can be used to configure a separate database for specific UCR data sources
+# The key is what you will reference in the datasource, e.g. 'custom_ucr_database'
+# The value is the sql connection string "postgresql://%(USER)s:%(PASSWORD)s@%(HOST)s:%(PORT)s/commcarehq_reporting" % DATABASES['default']
+CUSTOM_DATABASES = {}
```

Comment:
I think we should add this to `settings.py` so that it doesn't crash in `get_connection_string` if not defined